test_scripts/Defects/5_0/2481_ChoiceID_is_missing_in_PerformInteraction_response.lua

```diff
@@ -0,0 +1,144 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2481
+-- Description:
+-- 1) ChoiceID is missing in PerformInteraction response
+-- Precondition:
+-- 1) Vehicle Ignition On and Running
+-- 2) Android connected via BT
+-- 3) SyncProxyTester Media App running on phone and SYNC
+-- 4) Created Interaction Choice Set with ID 1 and data items as A, B and C respectively
+-- Steps to reproduce:
+-- 1) Select Send Message -> PerformInteraction and enter following data:
+--    Set the Initial text as 'Pick an Alphabet:', Initial Prompt as 'Pick a command', Interaction Mode as 'BOTH',
+--    Choice Set ID as '2', Check help prompt, timeout prompt and set timeout as 10000. Uncheck the VR help Item #. Press Ok.  
+-- Expected:
+-- 1) PerformInteraction(request) notificaiton is displayed on the app.
+--    Sync displays 'Pick an Alphabet' followed by the choice set A,B and C. Sync TTS 'Pick a command'
+-- 2) PerformInteraction(response): SUCCESS is displayed on the mobile app. Verify the ChoiceID in the response was correct.
+--    The app should next exit out of the PerformInteraction popup back to previous screen.
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+local paramsBoth = {
+    initialText = "Pick an Alphabet",
+    initalPrompt = "Pick a command",
+    interactionMode = "BOTH",
+    interactionChoiceSetIDList = { 1 }
+}
+
+-- [[ Local Functions ]]
+local function setExChoiceSet(choiceIDValues)
```

Comment:
interactionChoiceSetIDList contains only one value, so params can be defined without  additional function.